src/herder/TransactionQueue.cpp

```diff
@@ -53,47 +79,146 @@ TransactionQueue::tryAdd(TransactionFramePtr tx)
         return TransactionQueue::AddResult::ADD_STATUS_TRY_AGAIN_LATER;
     }
 
-    auto info = getAccountTransactionQueueInfo(tx->getSourceID());
+    int64_t netFee = tx->getFeeBid();
```

Comment:
I think we need to move the code that bans if we've reached the limit later so that we can properly deal with the net increase in operations when we replace by fee.